src/js/views/maps/viewfinder/ViewfinderView.js

```diff
@@ -77,14 +84,42 @@ define(
         return this.$el.find(`.${CLASS_NAMES.searchView}`);
       },
 
+      /**
+       * Helper function to focus input on the search query input and ensure
+       * that the cursor is at the end of the text (as opposed to the beginning
+       * which appears to be the default jQuery behavior).
+       */
```

Comment:
```suggestion
      /**
       * Helper function to focus input on the search query input and ensure
       * that the cursor is at the end of the text (as opposed to the beginning
       * which appears to be the default jQuery behavior).
       * @since x.x.x
       */
```